buildchow/templates/aggregate-site-template.yaml

```diff
@@ -0,0 +1,75 @@
+- job-template:
+    name: 'jbosstools-build-sites.aggregate.site_{stream}'
+    disabled: '{obj:disabled}'
+    description: |
+                 Tycho-based build, monitoring GitHub for changes.
+                 
+                 <p><i>Build should take ~60min.</i>
+                 
+                 <h2><a style="color:#FF9933" href="http://download.jboss.org/jbosstools/builds/cascade/">Build Job Cascade &amp;amp; Results</a></h2>
+
+    node: "{nodes}"
+    logrotate:
+      daysToKeep: 5
+      numToKeep: 5
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+    jdk: jdk1.7
```

Comment:
so aggregating p2 sites content (basically reading/genreating metadata and zipping up files)  require one to run the highest Java version ? That seems wrong. very wrong.

what if a component says it needs Java 6 wont run/start with Java 8 like jdt annotation bundle ?